packages/dds/tree/src/simple-tree/api/tree.ts

```diff
@@ -542,6 +548,20 @@ export interface TreeViewAlpha<
 
 	// Override the base branch method to return a typed view rather than merely a branch.
 	fork(): ReturnType<TreeBranch["fork"]> & TreeViewAlpha<TSchema>;
+
+	/**
+	 * Run a transaction which applies one or more edits to the tree as a single atomic unit.
+	 * @param transaction - The transaction callback to run.
+	 * @param params - The optional parameters for the transaction.
+	 */
+	runTransaction<TSuccessValue, TFailureValue>(
```

Comment:
Both of these overloads need a more complete doc comment.

Reply:
Added doc comments. Please take a look.